examples/webgl_depth_texture.html

```diff
@@ -0,0 +1,258 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>three.js webgl - Depth Texture</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <style>
+      body {
+        color: #fff;
+        font-family:Monospace;
+        font-size:13px;
+        text-align:center;
+        background-color: #000;
+        margin: 0px;
+        overflow: hidden;
+      }
+      a {
+        color: #B2E9FF;
+        font-weight: bold;
+        pointer-events: auto;
+      }
+      
+      canvas {
+        position: absolute;
+        top: 0;
+        left: 0;
+      }
+
+      #info {
+        pointer-events: none;
+        position: absolute;
+        left: 0;
+        top: 0px; width: 100%;
+        padding: 5px;
+        display: inline-block;
+      }
+      
+      #error {
+        margin: auto;
+        margin-top: 40px;
+        display: block;
+        max-width: 400px;
+        padding: 20px;
+        background: #CE0808;
+      }
+    </style>
+    
+    <script id="post-vert" type="x-shader/x-vertex">
+      uniform mat4 projectionMatrix;
+      uniform mat4 modelViewMatrix;
+
+      attribute vec4 position;
+      attribute vec2 uv;
+      varying vec2 vUv;
+
+      void main() {
+        vUv = uv;
+        gl_Position = projectionMatrix * modelViewMatrix * position;
+      }
+    </script>
+    <script id="post-frag" type="x-shader/x-fragment">
+      precision mediump float;
+
+      varying vec2 vUv;
+      uniform sampler2D tDiffuse;
+      uniform sampler2D tDepth;
+      uniform float cameraNear;
+      uniform float cameraFar;
+
+      float readDepth (sampler2D depthSampler, vec2 coord) {
+        float cameraFarPlusNear = cameraFar + cameraNear;
+        float cameraFarMinusNear = cameraFar - cameraNear;
+        float cameraCoef = 2.0 * cameraNear;
+        return cameraCoef / (cameraFarPlusNear - texture2D(depthSampler, coord).x * cameraFarMinusNear);
+      }
+
+      void main() {
+        vec3 diffuse = texture2D(tDiffuse, vUv).rgb;
+        float depth = readDepth(tDepth, vUv);
```

Comment:
You can remove the custom readDepth function and instead use this:
```
#include <packing>
...
float depth = invClipZToViewZ( texture2D( depthSampler, coord ).x, cameraNear, cameraFar );
float linearClipZ = viewZToLinearClipZ( depth, cameraNear, cameraFar );

gl_FragColor.rgb = vec3( linearClipZ );
```

This will reconstruct the true z depth value in eye space using standardized functions.